src/material/core/style/_elevation.scss

```diff
@@ -89,3 +94,12 @@ $prefix: 'mat-elevation-z';
     $easing: $transition-timing-function) {
   transition: private-transition-property-value($duration, $easing);
 }
+
+@function compute-color-opacity($color, $opacity) {
```

Comment:
The function name needs to be prefixed with an underscore so that it isn't exposed outside of this file.